Sources/NIOIMAPCore/Grammar/Command/Command.swift

```diff
@@ -94,13 +94,13 @@ public enum Command: Hashable {
     case idleStart
 
     /// Copies each message in a given set to a new mailbox, preserving the original in the current mailbox.
-    case copy(LastCommandSet<SequenceSet>, MailboxName)
+    case copy(LastCommandSet<SequenceNumber>, MailboxName)
```

Comment:
@danieleggert IIRC a `SequenceSet` allows the representation `1:*`, whereas `SequenceNumber` is just the `1` or `*`. Assuming that's true, do we definitely want this change?

Reply:
I changed `LastCommandSet` to be generic over the type that’s in the `MessageIdentifierSet`, and it’s thus still `1:*` here.

```
public enum LastCommandSet<N: MessageIdentifier>: Hashable {
    case set(MessageIdentifierSetNonEmpty<N>)
    …
```

Hope that makes sense?

Reply:
Yep makes sense!